Setup/Model/ProductInstaller.php

```diff
@@ -153,6 +166,70 @@ private function createProtectionPlanProduct(AttributeSetInterface $attributeSet
         }
     }
 
+    /**
+     * Adds customizable options to the protection plan product
+     *
+     * @param Product $product
+     * @return void
+     * @throws SetupException
+     */
+    private function addOptionsToProtectionPlanProduct(Product $product)
+    {
+        try {
```

Comment:
This will throw and break the loop if any option fails to create. The other path is to move the try inside of the foreach to catch every exception, and continue the loop. Thoughts here?

Reply:
Since all of the options are needed we will keep it as is. If we fail to create any of the options we will break the loop and throw an exception 

cc: @scott-taylor-extend